modules/renter/download.go

```diff
@@ -295,13 +323,34 @@ func (r *Renter) addDownloadToChunkQueue(d *download) {
 			continue
 		}
 
+		// Calculate number of pieces to add to the download list.
+		// If the chunk's download speed should be sped up, the piece
+		// count will be raised above the minimum required.
+		// speedupchunk is true if it is the Nth last chunk where N is defined by `numFastChunks`.
```

Comment:
I'm using neither, and it is rendering ok for me. The comments are in-line with the rest of the block and (mostly) less than 80 characters long. Shall I change them to a block comment, or a concatenate them to a single line?